docs/content/end-users/connect-registry.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Connect to a Registry
+description: Configure Porter to authenticate and connect to a registry
+---
+
+Porter stores bundles in OCI (Docker) registries.
+Learn how to configure Porter to authenticate and connect to your registry.
+
+* [Authenticate to a Registry](#authenticate-to-a-registry)
+* [Unsecured Registry](#connect-to-an-unsecured-registry)
+* [Registry Secured with Self-Signed Certificates](#connect-to-a-registry-secured-with-self-signed-certificates)
+
+## Authenticate to a Registry
+
+Porter uses Docker's cached credentials to authenticate to a registry.
+Before running a Porter command that requires authentication, first run `docker login REGISTRY` to authenticate.
+For example, use `docker login` to authenticate to Docker Hub or `docker login ghcr.io` for GitHub Container Registry.
+
+## Connect to an Unsecured Registry
+
+An unsecured registry communicates using HTTP instead of HTTPS.
+Porter automatically uses HTTP to communicate with local registries running on the loopback IP address, such as localhost or 127.x.x.x.
+No further configuration is required.
+
+You can try this out by running a local Docker registry in a container:
+
+```bash
+# Run an unsecured Docker registry
+docker run --name registry -d -p 0.0.0.0:5001:5000 registry:2
+
+# Copy a bundle to it
+porter copy --source ghcr.io/getporter/examples/porter-hello:v0.2.0 --destination localhost:5001/hello:v0.2.0
+
+# Interact with the bundle
+porter explain localhost:5001/hello:v0.2.0
+```
+
+If the registry is hosted on an non-loopback ip address or a domain name, use the \--insecure-registry flag to allow connecting to the registry.
+Your Docker engine must also be [configured with any insecure registries](https://docs.docker.com/registry/insecure/).
```

Comment:
Maybe we can have a section that's the prerequisite for connecting to an insecure registry in this doc before `Connect to an Unsecure Registry`, so that we don't need to repeat these two sentences  in both sections
```
Your Docker engine must also be [configured with any insecure registries](https://docs.docker.com/registry/insecure/).
The \--insecure-registry may be specified for any command that accepts a \--reference flag.
```